app/src/main/java/com/wtmberlin/util/CoroutineViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.wtmberlin.util
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlin.coroutines.CoroutineContext
+
+var ui: CoroutineDispatcher = Dispatchers.Main
+var io: CoroutineDispatcher = Dispatchers.IO
+var background: CoroutineDispatcher = Dispatchers.Default
+
+abstract class CoroutineViewModel : ViewModel(), CoroutineScope {
```

Comment:
This implementation is already available with `viewModelScope` in `lifecycle:2.1.0-RC`